types/traverse/index.d.ts

```diff
@@ -103,7 +103,7 @@ interface StaticTraverse {
   clone<T>(obj: T): T;
 }
 
-interface TraverseContext {
+export interface TraverseContext {
```

Comment:
export here is not needed. there is an `export =` at the bottom.

Reply:
if you want it to be exported put it under a `namespace traverse` declaration.

Reply:
@mhegazy Thanks for the review. I've updated the PR. Put everything under `namespace traverse` and export a function instead of a variable.